async_test.go

```diff
@@ -547,14 +562,14 @@ func BenchmarkAsyncThroughputNetwork1024(b *testing.B) {
 	var reader, writer net.Conn
 	start := make(chan struct{}, 1)
 
-	l, _ := net.Listen("tcp", ":3000")
+	l, _ := net.Listen("tcp", ":")
```

Comment:
I think it wouldnt make a difference but we shoudl explicitly have this as ":0"

Reply:
Fixed